### PR TITLE
[Connector] source: Adding a S3 file connector

### PR DESCRIPTION
**Pull request by PMjoydas**

### CODE REVIEW

File: components/Python/src/s3_file_source/s3_file_source.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            aws_secret_access_key=self._params["aws_secret_access_key"],
+        )
+
+        file_path = os.path.join(self._params["parent_directory"], "s3_file_" + str(uuid.uuid4()))
+        client.download_file(self._params["bucket"], self._params["key"], file_path)
+
+        return file_path
+
+
+def parse_args():
```
Comment (liororama) [inline components/Python/src/s3_file_source/s3_file_source.py]: Remove all "main" stuff"

File: components/Python/src/s3_file_source/README.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (liororama) [inline components/Python/src/s3_file_source/README.txt]: Remove this file too

File: components/Python/src/s3_file_source/s3_file_source.py (hunk 1 above)
Comment (PMjoydas) [reply components/Python/src/s3_file_source/s3_file_source.py]: Removed main

File: components/Python/src/s3_file_source/README.txt (hunk 2 above)
Comment (PMjoydas) [reply components/Python/src/s3_file_source/README.txt]: Removed README.txt

**Pull request merged**